support/pygments/koka.py

```diff
@@ -25,19 +25,19 @@ class KokaLexer(RegexLexer):
 
     keywords = [
         'infix', 'infixr', 'infixl',
-        'module', 'import', 'as',
-        'pub', 'abstract',
+        'module', 'import', 'as', 'mod'
+        'pub', 'abstract', 'ctx'
```

Comment:
What does `ctx` mean as a keyword?

Reply:
https://www.microsoft.com/en-us/research/uploads/prod/2023/07/fiptree-tr-v3.pdf

First class constructor contexts. You can start at section 3 of the paper, if you just want to get the high level idea of it. If you want to read the rest of the paper, and not just section 3 I would start at the beginning.

Reply:
Interesting. I see it is used only in the dev branch, so I hadn't spotted it before. Shouldn't this keyword also be mentioned in the parser and lexer definitions?

Reply:
Yeah, I'll have to look at the parser to see how it is done. It's used both in a type context, and as a keyword in the creation of a constructor context.